modules/core/src/v2/trading/tradingAccount.ts

```diff
@@ -133,8 +141,16 @@ export class TradingAccount {
       .asCallback(callback);
   }
 
+  affirmations(): Affirmations {
+    return new Affirmations(this.bitgo, this.enterpriseId, this);
+  }
+
+  settlements(): Settlements {
+    return new Settlements(this.bitgo, this.enterpriseId, this);
+  }
+
   partners(): TradingPartners {
-    return new TradingPartners(this, this.bitgo);
+    return new TradingPartners(this.enterpriseId, this, this.bitgo);
```

Comment:
What order are you suggesting? `enterprise, account` makes the most sense to me since that's the order of all of our APIs. I agree it looks a little funny with `this` being the second parameter but I think it makes the most sense. Or are you talking about `this.bitgo`? Thx

Reply:
The Affirmations contructor takes (bitgo, enterpriseId, tradingAccount), the Settlements constructor takes (bitgo, enterpriseId, tradingAccount), but the TradingPartners constructor takes (enterpriseId, tradingAccount, bitgo).

Not a huge issue but it just stuck out to me.

Reply:
Ah I see, on it